test/Lachain.UtilityTest/TestUtils.cs

```diff
@@ -9,6 +9,8 @@
 using Lachain.Utility;
 using Lachain.Utility.Utils;
 using Nethereum.Util;
+using System.Diagnostics;
```

Comment:
It is better to revert this file to original code if we don't make any changes in it really